### DISCUSSION TITLE
payment getway stripe

### DISCUSSION THREAD

**Discussion by aimanndaus** [sanidhyy/duolingo-clone#6] [Q&A] [answered]
can you provider how to deploy payment getaway once deploy at vercel but after checkout done must to trigger that user already finish pay and redirect to online url not to return localhost. i hopefully you can provide video or etc to config the test mode payment in online domain.

**Comment by sanidhyy** [comment, marked as answer]
You can watch necessary steps from Antonio's video: https://youtu.be/dP75Khfy4s4?t=30913
at timestamp: *8:35:13*.
